### PR TITLE
Updated Readme for Sample Apps

### PR DESCRIPTION
**Pull request by microsoftShannon**
Removed default language from supporting site URLs.
Removed typo misdirecting /apps/ to /app/

### CODE REVIEW

File: samples/apps/copilot-chat-app/README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ functions that work together to construct each response.
 
 # Configure your environment
 Before you get started, make sure you have the following requirements in place:
-- [.NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
-- [Node.js](https://nodejs.org/en/download)
-- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install)
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Node.js](https://nodejs.org/)
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:18]: the links for node and Yarn don't go to the install/download pages. Was this intentional?

**Comment by adrianwyatt** [comment]
I don't see the change for "Removed typo misdirecting /apps/ to /app/"

**Review by adrianwyatt — DISMISSED** [review]

File: samples/apps/copilot-chat-app/README.md (hunk 1 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:18]: Yes, exploring the web site they had different structures in other languages, so it seemed best to provide a link to the root page and allow users to review and confirm the materials they are downloading.

**Comment by shawncal** [comment]
> I don't see the change for "Removed typo misdirecting /apps/ to /app/"

I found what @microsoftShannon is referring to... this branch had such a change, but someone else fixed it in main and it disappeared with the merge.

**Review by shawncal — APPROVED** [review]

**Pull request merged**